### ISSUE TITLE
(IR) copy robot steps and tests that only have renames

### ISSUE THREAD

**Issue by ndushay** [sul-dlss/preservation_robots#6]
The following two robots and their tests should more or less be copied over from sdr-preservation-core, per sul-dlss/preservation2017#45.  Possibly after #5

>**(5) verify-agreement** [code](https://github.com/sul-dlss/sdr-preservation-core/blob/master/lib/sdr_ingest/verify_agreement.rb)
>- These seem to be fairly self-explanatory and need no functional change. It is actually the admin policy that is being verified, not the agreement, so a name change could clarify that.
>- Recommend: no functional change; optional name change and code-references updated from agreement to APO. 
>
>**(6) complete-deposit** [code](https://github.com/sul-dlss/sdr-preservation-core/blob/master/lib/sdr_ingest/complete_deposit.rb)
>- This is where the Moab object is placed into the Moab directory. Not a clear name since "deposit" is not clearly defined in the context of "ingest", which has a couple more steps to go.
>- Recommend: no functional changes; would like to change name to create-moab for clarity.

**Comment by ndushay** [comment]
superseded by #12 and #13
